app/views/payments/add.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :title, "Saldo opwaarderen voor #{@user.name}- SOFIA" %>
+
+<%= content_tag :div, id: 'payment-add', class: 'container footer-padding', data: {user_credit: @user.credit} do %>
+  <div class="container footer-padding">
+    <div class="row py-4">
+      <div class="col-12">
+        <h1>
+          Saldo opwaarderen voor <%= @user.name %>
+        </h1>
+
+        <%= simple_form_for(@payment, wrapper: :horizontal_form) do |f| %>
+          <div class="col-12">
+            <p>
+              Voer hier het bedrag in dat je wilt inleggen. Hierna zal je worden doorgestuurd naar de betaalomgeving.
+              Let op: Het bedrag moet gescheiden worden door een punt. Gebruik geen komma!
+            </p>
+            <%= f.input :amount, as: :float, input_html: {min: 20,  "v-model": "paymentAmount"}, required: true, label: 'Bedrag'  %>
```

Comment:
Here I can still manually enter an amount less than 20 without an error being thrown/shown

Reply:
Making amountValid() also check if the value is at least €20 probably fixes the issue